api/types/types.go

```diff
@@ -155,3 +155,27 @@ type Info struct {
 	Name               string
 	Labels             []string
 }
+
+type ContainerState struct {
```

Comment:
ok with that @duglin, working on it even if I don't like it because it's far better thinking about refactoring runconfig/opts so it can be included in api/types instead of rewriting each field in that really big struct